provider_shopper/lib/screens/login.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+class MyLogin extends StatelessWidget {
+  final _usernameController = TextEditingController();
+  final _passwordController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: Container(
+          padding: EdgeInsets.all(70.0),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: <Widget>[
+              Text(
+                'Welcome',
+                style: Theme.of(context).textTheme.display4,
+              ),
+              TextFormField(
+                controller: _usernameController,
```

Comment:
While it can't be seen in the original GIF, some hints would be great here. Try adding something like this to each field:

```
decoration: InputDecoration(
  hintText: 'Username',
),
```